vitest.integration.ts

```diff
@@ -1,10 +1,15 @@
 import { defineConfig } from "vitest/config";
 import { baseConfig } from "./vitest.config";
 
+console.log(`
+Running integration tests...
+`);
+
 export default defineConfig({
   test: {
     ...baseConfig,
     include: ["**/*.test.slow.?(c|m)[jt]s?(x)"],
     testTimeout: 60000,
+    threads: false,
```

Comment:
If we try to use leveldb in separate processes, it breaks.

Reply:
Will this be a problem? How does the grpc server scale requests? Does it use multiple threads or just event loop?

Reply:
This shouldn't be a problem with gRPC since the server's a single process so we only instantiate leveldb once. The issue here was that vite splits up tests across multiple processes which were trying to use the same leveldb cache.